src/baseline_bots/utils.py

```diff
@@ -253,6 +253,39 @@ def parse_alliance_proposal(msg: str, recipient: str) -> List[str]:
         raise ParseError("A minimum of 2 powers are needed for an alliance")
 
 
+def parse_peace_proposal(msg: str, recipient: str) -> List[str]:
```

Comment:
In a perfect world, this code could be shared with alliance for less duplication, but probably don't have time for that

Reply:
Later this week, when the deadline is met, I'm going to spend a lot of time doing code cleanup and refactoring. The parsing code will be one of my focuses.

Reply:
I have been looking into an existing DAIDE-focused parsing library and plan to bring it up to the team as a whole soon.